docs/onboarding/03-sdks-and-apis.md

```diff
@@ -0,0 +1,34 @@
+# Chapter 3: Digital Twin SDK's and API's
+
+Azure digital twins provides a C# SDK for programmatic management of models and twins. The following covers the capabilities of the SDK's and APIs
+
+- [Use the API's and SDKs](https://docs.microsoft.com/en-us/azure/digital-twins/how-to-use-apis-sdks)
+- [Create Graph with APIs](https://docs.microsoft.com/en-us/azure/digital-twins/concepts-twins-graph#create-with-the-apis)
+
+## Key Points
+
+- The SDK allows programmatic management of models and twins as well as querying the twin graph.
+- The SDK is backed by Azure Digital Twins API. The API could be leveraged when a client SDK is not available.
+- The SDK authenticates using the Azure.Identity library (DefaultAzureCredentials preferred).
+- Twins can be represented in type-safe code as POCO's and JSON Serialization.
+
+## Learning Exercises
+
+Using the models created from [Learning Exercises in Chapter 2](02-digital-twin-definition-language.md), create a console app that is able to do the following operations.
+
+1. Creates a model
+1. Checks if a model exists before attempting to create it.

```

Comment:
Should 1 & 2 be switched, just logically?

Reply:
just consolidated these. after re-reading seems a bit too granular.